libexec/cli/list.help

```diff
@@ -0,0 +1,32 @@
+USAGE: singularity [...] list [list options...]
+
+List all running named instances
```

Comment:
Actually, it maybe possible for us to add a real `singularity ps ...` at some point which will properly show a ps like output for all processes running within a container. We may even be able to write a `top` like interface too!